### ISSUE TITLE
User guide for organizers.

### ISSUE THREAD

**Issue by AndrewVSutherland** [roed314/seminars#105]
We should add a document describing the workflow for seminar/conference organizers that includes a "getting started" section on how they can set up their seminar (including an explanation of features that might not be obvious at first glance, e.g. how talks inherit properties from seminars, how to manage the schedule layout, etc...), as well as a guide to the workflow, including sending links to speakers to let them post their abstracts and slides (either before or after the talk).

**Comment by poonen** [comment]
There is enough on the site that is self-explanatory, or explained by knowls, that I think most users would not bother to look at such a document.  My feeling is that it is better just to improve the explanations in the knowls and on the FAQ, if there are things that are confusing.

**Comment by edgarcosta** [comment]
agree

**Comment by AndrewVSutherland** [comment]
I agree that we should never need to depend on the existence of such a guide to clear up confusion. 
But there is always going to be a category of user who is nervous about clicking around and wants to understand everything in detail before they start, and there are also places where we can provide guidance and encourage behavior (e.g. suggesting that organizers embed our schedule in their site so they don't have to enter data twice and giving details and examples on how to do that) that I think will be more effective than simply assuming they will at some point click on the right knowl.

With so much of the UI in flux, there is no reason for us to work on this now, we should wait until things stabilize, so I'm going to mark it as future, but I don't think we should just close it.  I'm willing to be that if you polled our endorsed users a majority of them would vote in favor of creating such a document.  This will also be useful if/when we ever start to outsource user support to others.
